agent/agent_test.go

```diff
@@ -546,8 +546,8 @@ func TestRunnerRun(t *testing.T) {
 			Recipe:      validRecipe,
 			RecordCount: 1,
 			Success:     true,
-			// DurationInMs is 0 because of the process took below a millisecond, in production scenario it takes longer than that
-			DurationInMs: 0,
+			// DurationInMs is 1000 as an arbitary alloted duration time, in production scenario it takes longer than that
+			DurationInMs: 1000,
```

Comment:
let's store this `1000` as a variable e.g. `expectedDuration` instead of hardcoding it here and line 564 below